src/build-extensions-oscontainer.sh

```diff
@@ -2,16 +2,17 @@
 #Used by cmd/build-extensions-container.go
 #Find the RHCOS ociarchive.
 set -euo pipefail
-buildid=$1
+arch=$1
 shift
 filename=$1
+buildid=$2
```

Comment:
I more meant

```suggestion
filename=$1
shift
buildid=$1
```

But I don't have a really strong opinion; this whole thing could be
```
arch=$1
filename=$2
buildid=$3
```
too if you prefer.

Reply:
yeah on the first commit I had not seen your comment. Should be like this now.